### PR TITLE
Add deprecation message to the browser's console

### PR DESCRIPTION
**Pull request by terjanq**
As of January 31st, 2021, the Caja project is going to be deprecated.

### CODE REVIEW

**Comment by terjanq** [comment]
CC @kpreid

File: src/com/google/caja/plugin/caja.js [hunk 1]
```diff
@@ -23,6 +23,29 @@
...
+  // Print a note about it in the browser's console
+  var deprecation_message = [
+    'On January 31st, 2021, we will be archiving the Caja project. ',
+    'After January 31, no new features will be added, pull requests and other ',
+    'issues will no longer be addressed, including patches for security issues, ',
+    'and the repository will be marked as archived. Caja has not been actively ',
+    'maintained or developed to keep up with the latest research on web security. ',
+    'As a result, several security vulnerabilities have been reported to Caja, ',
+    'both by Google’s security engineers and by external researchers.\n\n',
+    'We encourage Caja users to migrate to Closure toolkit ',
```
Comment (kpreid) [inline src/com/google/caja/plugin/caja.js:36]: Anyone who is loading `caja.js` (as opposed to `html-sanitizer.js`) is quite likely intending to support embedded JavaScript. "Migrate to Closure's sanitizers" is not a replacement for those use cases. Please rephrase this to at least *acknowledge* that this recommendation is only relevant to non-script-containing content, whether or not there is any recommendation for a replacement.

Comment (kpreid) [reply src/com/google/caja/plugin/caja.js:36]: P.S.: this _would_ be an appropriate message to embed in `html-sanitizer.js` and `html-sanitizer-legacy.js`.

**Comment by terjanq** [comment]
I applied the change in the message. Let me know if that works.

**Review by kpreid — APPROVED** [review]

**Pull request merged**